### PR TITLE
Icons for accessories-ebook-reader

### PR DESCRIPTION
**Pull request by maxtron95**
accessories-ebook-reader-ci
accessories-ebook-reader-sq

### CODE REVIEW

**Review by palob — APPROVED** [review]
design is alright, what's the matter with data.json? I'm failing to see it.

**Review by Foggalong — APPROVED** [review]
@palob I'm currently trying to debug this but I _think_ it's on the end of the test scripts / Travis rather than anything @creepertron95 has done. I'm also fine with the design :+1:

**Comment by Foggalong** [comment]
@bilelmoussaoui As our resident Travis/Docker guru, any insight as to what is happening here?

File: data.json [hunk 1]
```diff
@@ -188,6 +188,11 @@
             ]
         }
     },
+    "accessories-ebook-reader": {
+        "linux": {
+            "accessories-ebook-reader"
```
Comment (bilelmoussaoui) [inline data.json]: This is an object and not a list it should be `"root" : "accessories-ebook-reader"`

Comment (palob) [reply data.json]: Oh yeah, all looking too similar to stand out

**Review by palob — APPROVED** [review]

File: data.json (hunk 1 above)
Comment (Foggalong) [reply data.json]: :facepalm: thanks for pointing that out @bilelmoussaoui, clearly we couldn't see the wood for the trees

**Pull request merged**